src/libuvc/libuvc.h

```diff
@@ -7,7 +7,11 @@ extern "C" {
 
 #include <stdio.h> // FILE
 #include <errno.h>
+#if defined(RS_USE_V4L2_BACKEND)
```

Comment:
ok. let me remove this patch, and try to fix it in `libuvc`
btw, `pkg-config` gives me this location
```
> pkg-config --cflags libusb-1.0
-I/usr/include/libusb-1.0
```